app/src/main/java/com/shippedsuite/example/MainFragment.kt

```diff
@@ -29,6 +30,13 @@ class MainFragment : Fragment() {
 
     private val defaultOrderValue = BigDecimal.valueOf(129.99)
 
+    private val configuration = ShippedSuiteConfiguration(
+        ShippedSuiteType.SHIELD,
+        false,
+        true,
+        true
+    )
```

Comment:
is there an alternative instantiation that is more descriptive?, e.g. 

```
ShippedSuiteConfiguration(
  type: ShippedSuiteType.SHIELD,
  isInformational: false,
  isMandatory: true,
  isRespectServer: true
)
```

just because it's easier to comprehend.

Reply:
@stevenou Actually it looks like this in `Android Studio`.

<img width="456" alt="Screenshot 2023-03-03 at 10 34 20" src="https://user-images.githubusercontent.com/3202811/222617235-78dedd2f-90fb-44fd-af56-01f1385dfded.png">

It can also be initialized like this:

<img width="462" alt="Screenshot 2023-03-03 at 10 35 37" src="https://user-images.githubusercontent.com/3202811/222617378-feafdbae-a32f-4118-aee0-d55fde1c2c18.png">